### ISSUE TITLE
`AutomationLiveSetting` value not handled by `TextBlockAutomationPeer` 

### ISSUE THREAD

**Issue by vgromfeld** [microsoft/microsoft-ui-xaml#7762]
### Describe the bug

To prevent screen readers from notifying too much the users when something changes, we can use the [AutomationLiveSetting](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationlivesetting?view=winrt-22621) enumeration values with the [AutomationProperties.LiveSetting](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.automationproperties.livesetting?view=winrt-22621) attached property.

Setting this value to `Off` or `Polite` does not have any effect on `TextBlock` controls.
When the refresh rate of the `TextBlock` content is shorter than the time needed to say it, Windows' narrator keeps saying the beginning of the content.

### Steps to reproduce the bug

1. Create a new UWP app with the following XAML and code behind
```xml
<Grid>
    <!-- Both Off or Polite values are ignored. -->
    <TextBlock x:Name="textblock" AutomationProperties.LiveSetting="Off" />
</Grid>
```

```cs
protected override void OnNavigatedTo(NavigationEventArgs e)
{
    _timer = new DispatcherTimer
    {
        Interval = TimeSpan.FromSeconds(1),
    };

    _timer.Tick += OnTimerTick;
    _timer.Start();
}

private void OnTimerTick(object sender, object e)
{
    _counter++;
    textblock.Text = $"Counter value is {_counter}";
}
```
2. Run the app
3. Turn on Windows narrator using CTRL+WIN+ENTER
4. Use CAPS LOCK+LEFT/RIGHT to highlight the text
5. The narrator will say "Counter value" in loop.

### Expected behavior

The narrator should finish what it has to say before reading the new value or says nothing if the live region is disabled.

### Screenshots

![image](https://user-images.githubusercontent.com/20152272/192284068-5e9f413c-ca7b-4187-81f8-f6696d0cf4a7.png)

### Windows app type

- [X] UWP
- [ ] Win32

### Device form factor

Desktop

### Additional context

The repro app: [TextBlockAutomationPeer.zip](https://github.com/microsoft/microsoft-ui-xaml/files/9646431/TextBlockAutomationPeer.zip)
